src/Infrastructure/WB.Persistence.Designer/Migrations/M2022/M202204111809_AddAnonymousQuestionnaireTable.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Data;
+using FluentMigrator;
+
+namespace WB.UI.Designer.Migrations.PlainStore
+{
+    [Migration(2022_04_11_18_09)]
+    public class M202204111809_AddAnonymousQuestionnaireTable : ForwardOnlyMigration
+    {
+        public override void Up()
+        {
+            this.Create.Table("anonymous_questionnaires")
+                .WithColumn("anonymous_questionnaire_id").AsGuid().PrimaryKey()
+                .WithColumn("questionnaire_id").AsGuid().NotNullable()
```

Comment:
Please add index for "questionnaire_id" column. Postgres doesn't automatically create index for foreign keys but we are doing queries to this table with condition on this column (plus delete cascade rule)